### DISCUSSION TITLE
Script to update PKGBUILDs

### DISCUSSION THREAD

**Discussion by acxz** [rocm-arch/rocm-arch#971] [General]
Hello all, long time no see, but I've found some time to give back for a bit. Was doing some clean up and found a nifty script I made, thought I'd share it. Might be most relevant to @tpkessler.

https://gist.github.com/acxz/bc40b850be614e9d09e6066ba4f1b430

As its name implies, the script is a bash script that clones the repo, updates the PKGBUILDS to a specified version that can be built, pushes to a branch on this repo and cleans up.

Could probably make the specified version (`latest_version` in the script) be read in from the command line, but otherwise it is self contained and automagically script for just version bumps.

Definitely not the worst bash user, but was able to speed my development tremendously with the help of our new friend ChatGPT.

**Comment by tpkessler** [comment]
Nice script! With the new tooling for Arch packages (`pkgctl` in `devtools`) you can do the same with one command:
```
pkgctl build --release --db-update --pkgver=5.7.0
```
This pushes the changes directly to the official mirrors (if you have access). Otherwise, I find it very useful to build AUR packages in a clean chroot:
```
pkgctl build --repo extra --pkgver=5.7.0
```

**Reply by acxz** [reply]
Wow that's lit, def gonna have to check out `pkgctl`, thx for sharing!
